tests/datasets/custom.py

```diff
@@ -111,3 +111,33 @@ def test_fetch_data_save(self):
         self.assertTrue(
             Path(dataset._save_path).is_dir(),
         )
+
+    def test_fetch_index_data(self):
+        """ """
+        dataset = CustomDataset(
+            nasdaq_index="OMXS30",
```

Comment:
Good we are testing with the smallest index as of now, since its not mocked. 
We should really try and get the mocking up and running, so we can get full coverage on
all our defined indexes :)